### PR TITLE
feat(): add topic about angular cache

### PR DESCRIPTION
**Pull request by eduardoRoth**
Add a topic about Angular Cache (new feature from v13) and what you need to do is updating from previous versions.

### CODE REVIEW

**Comment by d3lm** [comment]
Cool, but this content does not match the general structure of Problem -> Solution. Maybe you can elaborate a bit and first explain the problem and then present the solution (which is probably to use the new cache).

**Comment by eduardoRoth** [comment]
Sure, I'll update this today too

**Comment by eduardoRoth** [comment]
@d3lm updated!

File: content/performance/use-angular-new-cache.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+The build process on large Angular applications can take long time for it to complete, and having to rebuild the application frequently will take a toll at the time you spend waiting for it to finish.
+
+# Solution
+###### **This only applies to Angular v13+**
+
+Use the new persistent build cache.
+
+From [Angular v13 is now available](https://blog.angular.io/angular-v13-is-now-available-cce66f7bc296) official Angular Blog post:
+> The valuable feedback from [[RFC] Persistent build cache by default](https://github.com/angular/angular-cli/issues/21545) led to this tooling update that results in up to 68% improvement in build speed and more ergonomic options.
```
Comment (d3lm) [inline content/performance/use-angular-new-cache.md:15]: Newline before.

File: content/performance/use-angular-new-cache.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        }
+    }
+    ...
+}
+```
+
+
+# Resources
+
+- [Angular Cache](https://angular.io/cli/cache) - Official documentation for `Angular Cache`
```
Comment (d3lm) [inline content/performance/use-angular-new-cache.md:36]:
```suggestion
- [Official documentation for Angular's persistent disk cache](https://angular.io/cli/cache)
```

File: content/performance/use-angular-new-cache.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+{
+    "$schema": "...",
+    "cli": {
+        "cache": {
+            "enabled": true,
+            "path": ".cache",
+            "environment": "all"
+        }
+    }
+    ...
```
Comment (d3lm) [inline content/performance/use-angular-new-cache.md:29]: Indentation should be 2 spaces.

File: content/performance/use-angular-new-cache.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+# Solution
+###### **This only applies to Angular v13+**
+
+Use the new persistent build cache.
+
+From [Angular v13 is now available](https://blog.angular.io/angular-v13-is-now-available-cce66f7bc296) official Angular Blog post:
+> The valuable feedback from [[RFC] Persistent build cache by default](https://github.com/angular/angular-cli/issues/21545) led to this tooling update that results in up to 68% improvement in build speed and more ergonomic options.
+
+New projects using Angular CLI v13 have cache already enabled by default but, if you're updating your app from previous versions, you need to add the following to you `angular.json` file:
```
Comment (d3lm) [inline content/performance/use-angular-new-cache.md:17]:
```suggestion
New projects using Angular CLI v13+ have the persistent disk cache already enabled by default but, if you're updating your app from previous versions, you need to add the following to you `angular.json` file:
```

File: content/performance/use-angular-new-cache.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+# Problem
+
+The build process on large Angular applications can take long time for it to complete, and having to rebuild the application frequently will take a toll at the time you spend waiting for it to finish.
+
+# Solution
+###### **This only applies to Angular v13+**
+
+Use the new persistent build cache.
+
+From [Angular v13 is now available](https://blog.angular.io/angular-v13-is-now-available-cce66f7bc296) official Angular Blog post:
```
Comment (d3lm) [inline content/performance/use-angular-new-cache.md:14]:
```suggestion
From [Angular v13 is now available](https://blog.angular.io/angular-v13-is-now-available-cce66f7bc296):
```

**Review by d3lm — CHANGES_REQUESTED** [review]
Thanks for the PR! Left some remarks but it's a great addition 👏

File: content/performance/use-angular-new-cache.md (hunk 1 above)
Comment (eduardoRoth) [reply content/performance/use-angular-new-cache.md:15]: @d3lm Should I add one more line or remove the one that's there?

Comment (d3lm) [reply content/performance/use-angular-new-cache.md:16]: Add on between the quote and `From...`, so
```diff
 From [Angular v13 is now available](https://blog.angular.io/angular-v13-is-now-available-cce66f7bc296) official Angular Blog post:
+ 
 > The valua...
```

**Comment by eduardoRoth** [comment]
@d3lm it's ready to be merged.

**Review by d3lm — APPROVED** [review]

**Pull request merged**